pyproject.toml

```diff
@@ -92,7 +92,8 @@ jit = "duplo_resource.jit:DuploJit"
 system = "duplo_resource.system:DuploSystem"
 asg = "duplo_resource.asg:DuploAsg"
 secret = "duplo_resource.secret:DuploSecret"
-config = "duplo_resource.configmap:DuploConfigMap"
+configmap = "duplo_resource.configmap:DuploConfigMap"
```

Comment:
This change will break some of our customers who are using just `config` in their pipelines. What are your thoughts? I think `configmap` is more proper. This comment is more of a thought. 

Two features that could help here. 
1. alias feature where we could use both
2. warnings like 'did you mean configmap?'

We can discuss on these later.